RadixWallet/Features/CardCarousel/CardCarousel+View.swift

```diff
@@ -0,0 +1,211 @@
+import Foundation
+import SwiftUI
+
+// MARK: - CardCarousel.View
+extension CardCarousel {
+	public struct View: SwiftUI.View {
+		public let store: StoreOf<CardCarousel>
+
+		private static let coordSpace: String = "CardCarousel"
+
+		private let margin: CGFloat = .medium1
+		private let spacing: CGFloat = .small1
+		@ScaledMetric private var height: CGFloat = 105
+
+		public var body: some SwiftUI.View {
+			WithPerceptionTracking {
+				GeometryReader { proxy in
+					coreView
+						.backgroundPreferenceValue(PositionsPreferenceKey.self) { positions in
+							dummyCards(positions, in: proxy.frame(in: .named(Self.coordSpace)))
+						}
```

Comment:
I am not really sure about the purpose of these dummy cards in the background, since the height is already set to 105 (dynamically changing based on device font size configuration).

As these videos show, adding the dummy views presents a glitch that would be avoided if removed:
| With dummy views | Without dummy views |
| -- | -- |
| <video src=https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/35b3ac6e-5a67-4f54-bf5b-b6443fbe3381> | <video src=https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/cba6c06a-8664-424a-9153-72accd45af55> |
| <video src=https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/c7280444-3c96-4f4d-8cef-5fca79c42380> | <video src=https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/ecd53fb4-e999-4bea-a157-f03015357974> |